src/import-data/ImportData.ts

```diff
@@ -87,119 +103,133 @@ export const ImportData = () => {
     loadTemplateWorkspaces();
   });
 
-  const importPFB = (namespace, name) => {
-    return async () => {
-      const { jobId } = await Ajax().Workspaces.workspace(namespace, name).importJob(url, 'pfb', null);
-      asyncImportJobStore.update(Utils.append({ targetWorkspace: { namespace, name }, jobId }));
-      notifyDataImportProgress(jobId);
-    };
+  const importPFB = async (importRequest: PFBImportRequest, workspace: WorkspaceInfo) => {
+    const { namespace, name } = workspace;
+    const { jobId } = await Ajax().Workspaces.workspace(namespace, name).importJob(importRequest.url, 'pfb', null);
```

Comment:
Since `pfb` is in `type` you could use that here
```suggestion
    const { jobId } = await Ajax().Workspaces.workspace(namespace, name).importJob(importRequest.url, importRequest.type, null);
```

Maybe the import methods should instead accept the more specific request types, such as `importBagit(BagItImportRequest)`?

Reply:
I didn't want to make any changes to the Ajax functions in this PR. Currently, the Ajax functions generally have very little abstraction over what's passed to the API endpoint.